YoCode/DuplicationCheck.cs

```diff
@@ -43,6 +44,14 @@ public DuplicationCheck(PathManager dir, string CMDtoolsDirConfig)
             origArguments = Path.Combine(dir.originalTestDirPath, fileNameChecked) + outputArg + outputFile;            
             
             ExecuteTheCheck();
+            }
+            catch(Exception e)
+            {
+            DuplicationEvidence.FeatureImplemented = false;
+            DuplicationEvidence.GiveEvidence("To run this application you will have to install Command Line Tools by Jetbrains\n" +
+                "Direct download link here: https://www.jetbrains.com/resharper/download/download-thanks.html?platform=windows&code=RSCLT" +
+                "After you downloaded it please specify its location in appsetting.json file, which lives in the root directory of this  project ");
```

Comment:
Rather than duplicate this string. make a class that has it as a constant that both places can reference.

Reply:
This should be done as another issue as well I think